### PR TITLE
[Github] Update the pull-request template

### PR DESCRIPTION
**Pull request by sroze**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | ø
| License       | MIT
| Doc PR        | ø

2.7 is end of life.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | master for features / 2.7 up to 4.0 for bug fixes <!-- see below -->
+| Branch?       | master for features / 2.8 up to 4.0 for bug fixes <!-- see below -->
```
Comment (jvasseur) [inline .github/PULL_REQUEST_TEMPLATE.md]: `4.0` can be replaced with `4.1` at the same time.

Comment (sroze) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good point. Changed 😉

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
4.0 is maintained until July 2018, right?

**Comment by sroze** [comment]
@fabpot sure, [the master piece](http://symfony.com/doc/current/contributing/community/releases.html#schedule) agrees; but if we say "up to" we need to mention the upper branch that isn't master I think. And it's 4.1 now :)

**Review by nicolas-grekas — APPROVED** [review]

**Review by ogizanagi — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @sroze.

**Pull request merged**